### DISCUSSION TITLE
Show sum of imports and exports

### DISCUSSION THREAD

**Discussion by wastlmuc** [electricitymaps/electricitymaps-contrib#5306] [Ideas] [answered]
Hi all,

currently the different zones/countries have their respective value of imported/exported energy in the chart. Would it be possible to have an option to show the sum of import and export?

We are facing quite some discussions in Germany, on this matter, but a lumped sum would probably help in this context.

Kindly let me know what you think.
Wastl

**Comment by VIKTORVAV99** [comment]
Hi!

We have started working on shaping a feature that should cover some of these needs in #5259. This should cover the sum of exports at least.

We are also thinking of adding a graph that shows the net import/export like our other graphs but we have not decided anything about this yet but I wanted to at least provide a rough status of it. I'll make note of this request though and see if I can get it added to next internal roadmap.

**Comment by wastlmuc** [comment]
Hi,

This sounds great!

Best
Wastl

Am 21. April 2023 22:17:48 MESZ schrieb Viktor Andersson ***@***.***>:
>Hi!
>
>We have started working on shaping a feature that should cover some of these needs in #5259. This should cover the sum of exports at least.
>
>We are also thinking of adding a graph that shows the net import/export like our other graphs but we have not decided anything about this yet but I wanted to at least provide a rough status of it. I'll make note of this request though and see if I can get it added to next internal roadmap.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/electricitymaps/electricitymaps-contrib/discussions/5306#discussioncomment-5689854
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>

Sent from mobil.

**Comment by madsnedergaard** [comment, marked as answer]
@VIKTORVAV99 implemented the net exchange graph in #5647, so you can now see total import/export for any zone over time :)
